### PR TITLE
fix(admin-ui): Make copy on manage locations modal better when no locations

### PR DESCRIPTION
**Pull request by StephixOne**
Updated the copy and UI on the manage locations modal on a variant better when there are no locations available

Before:
image

Now:
image

Resolves CORE-1295

### CODE REVIEW

**Review by olivermrbl — APPROVED** [review]
LGTM - please add a changeset :)

File: packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx [hunk 1]
```diff
@@ -342,24 +342,39 @@ const ManageLocationsForm = ({
...
+            <div
+              className={clsx(
+                "border-grey-20 text-grey-50 flex w-full items-center justify-between",
+                { "pb-base border-b": locationsAvailable }
+              )}
+            >
+              <div>
+                <p>
+                  {locationsAvailable
+                    ? "Select locations that stock the selected variant"
```
Comment (adrien2p) [inline packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx:361]: nit: what about `Select the location for the selected variant stock`

**Comment by StephixOne** [comment]
> LGTM - please add a changeset :)

Done, was just exploring something extra 😁

File: packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx [hunk 2]
```diff
@@ -342,24 +342,39 @@ const ManageLocationsForm = ({
...
+                    : "You don't have any locations"}
```
Comment (adrien2p) [inline packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx:362]: suggestion: singular? wdyt?

**Review by adrien2p — APPROVED** [review]
LGTM, some nit comment if acepted

File: packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx (hunk 1 above)
Comment (StephixOne) [reply packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx:361]: You can select multiple, if you have them, and imo that's better copy in this case.

File: packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx (hunk 2 above)
Comment (StephixOne) [reply packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx:362]: As above, the plural is better copy imo, as you can have multiple. You'd say "You do not have a stock location" if you could only ever have one.

File: packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx (hunk 1 above)
Comment (adrien2p) [reply packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx:361]: Oups sorry you right 🤘

File: packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx (hunk 2 above)
Comment (adrien2p) [reply packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx:362]: For example would you « you don’t have any car » or « you don’t have any cars »? If that make sense

Comment (olivermrbl) [reply packages/admin-ui/ui/src/components/forms/product/variant-inventory-form/variant-stock-form/index.tsx:362]: I am also in favor of the plural version

**Pull request merged**